src/include/kernel/ext_manifest.h

```diff
@@ -95,7 +96,7 @@ struct ext_man_dbg_abi {
 struct ext_man_config_data {
 	struct ext_man_elem_header hdr;
 
-	struct config_elem elems[EXT_MAN_CONFIG_LAST_ELEM];
+	struct config_elem elems[EXT_MAN_CONFIG_LAST_ELEM - EXT_MAN_CONFIG_FIRST_ELEM];
```

Comment:
Hmm, starts to be a convoluted to me. I didn't pay attention when the "LAST_ELEM" was added, so without context of the prior discussions, I'd say simple "struct config_elem elems[EXT_MAN_CONFIG_LAST_ELEM - 1];" would be simple enough.

Reply:
@kv2019i This would be easy for view and later change. You will no longer need to update the `1` to other values if you change the ENUM or add some comments here to say: "the enum starts at 1"